### ISSUE TITLE
Cannot read property 'roles' of undefined

### ISSUE THREAD

**Issue by briantanner** [hydrabolt/discord.js#324]
This is happening on indev on occasion, uninitiated.

```
/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/lib/Client/InternalClient.js:2092`
								roles: server.memberMap[data.user.id].roles,
								                                     ^

TypeError: Cannot read property 'roles' of undefined
    at WebSocket.websocket.onmessage (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/lib/Client/InternalClient.js:2092:46)
    at WebSocket.onMessage (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/WebSocket.js:442:14)
    at emitTwo (events.js:100:13)
    at WebSocket.emit (events.js:185:7)
    at Receiver.ontext (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/WebSocket.js:841:10)
    at /Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:536:18
    at Receiver.applyExtensions (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:371:5)
    at /Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:508:14
    at Receiver.flush (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:347:3)
    at Receiver.opcodes.1.finish (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:541:12)
    at Receiver.expectHandler (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:499:31)
    at Receiver.add (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/Receiver.js:103:24)
    at TLSSocket.realHandler (/Users/briantanner/GitRepos/Grepolis-DiscordBot/node_modules/discord.js/node_modules/ws/lib/WebSocket.js:825:20)
    at emitOne (events.js:90:13)
    at TLSSocket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18)
```

**Comment by brussell98** [comment]
I've also been getting this for a while. Mine says `TypeError: Cannot read property 'roles' of null`
Happening at `InternalClient.js:2092:46`

**Comment by abalabahaha** [comment]
Hopefully fixed by af333ef
